src/styles/themes/_dark.scss

```diff
@@ -41,7 +41,8 @@
   --color-background-alternative-deep: var(--shade-5);
 
   --color-background-inverse: #fff;
-  --color-background-backdrop: rgba(255, 255, 255 0.25);
+  --color-background-backdrop: rgba(255, 255, 255, 0.25);
+  --color-background-backdrop-inverted: rgba(0, 0, 0, 0.8);
```

Comment:
hmmm, can we use --color-background-lighten: rgba(0, 0, 0, 0.75); instead of creating inverted?